web/src/stream_settings_ui.js

```diff
@@ -797,7 +798,7 @@ export function switch_rows(event) {
     const row_data = get_row_data($switch_row);
     if (row_data) {
         const stream_id = row_data.id;
-        switch_to_stream_row(stream_id, "general");
+        switch_to_stream_row(stream_id, stream_edit_toggler.select_tab);
```

Comment:
Ah oops, I changed that last time for some reason assuming that's how it was supposed to work. Sorry about that, and thanks for fixing it back!